mars/services/task/supervisor/manager.py

```diff
@@ -40,20 +40,20 @@ class TaskConfigurationActor(mo.Actor):
     def __init__(
         self,
         task_conf: Dict[str, Any],
-        task_executor_config: Dict[str, Any],
+        execution_config: Dict[str, Any],
```

Comment:
Is the type right?

Reply:
The type here is right. The `TaskExecutor` accepts the execution config in dict type and converts it to the correct `ExecutionConfig` object according to the backend. e.g. The `RayTaskExecutor` will be constructed with a `RayExecutionConfig`.